### DISCUSSION TITLE
How to dispatch a redux action

### DISCUSSION THREAD

**Discussion by SeongwoonHong** [pmndrs/zustand#898] [Q&A] [answered]
Hello.

little background, i'm using Zustand for a feature A, and I use redux for global state management in my app.

so here's an example code

```
const zustandState = (set: SetState, get: GetState) => ({
  plan: {},
  zustandPeople: async (something) => {
     // here I want to dispatch an action that's defined in redux globally.
  },

});
const createStore = create(zustandState)

export const useSomethingStore = createStore;

```

And imagine, there's a People redux store and if you dispatch an action "GET_PEOPLE" it calls a relevant api and saves the response to People in redux store.
but when I call "zustandPeople" function, i want to do the same thing for some reason.
how do I do that? I know there's a redux middleware but it seems to only work with the specific reducer you pass to redux middleware?

**Comment by dai-shi** [comment, marked as answer]
```js
  zustandPeople: async (something) => {
     // you can dispatch an action as you have redux `store` globally.
    store.dispatch({ type: 'GET_PEOPLE' })
  },
```
